sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -491,6 +548,4 @@ async def write(self, s):
         See http://tools.ietf.org/html/rfc5234
         http://tools.ietf.org/html/rfc6455#section-5.2
         """
-        await self.loop.run_in_executor(
-                None, self.ws.send_binary, s
-                )
+        await self.ws.send_bytes(s)
```

Comment:
Not super familiar with this, but is there a way we could patch send() to send_bytes?

Reply:
Im not sure I follow, are you saying add in a send in to  `WebSocketTransportAsync`?

Reply:
Sorry I think I got confused with the context manager response versus the documentation. I meant if what if you call self.ws.send() instead of send_bytes() and would that send the information in bytes?

Reply:
ah gotcha, `ClientWebSocketResponse` returned by `aiohttp.ws_connect` doesn't have `send` but `send_str`, `send_json` and `send_bytes`.  It does have a receive method which does what you are suggesting :)